pytorch/edgeml_pytorch/graph/rnn.py

```diff
@@ -967,78 +967,142 @@ class BaseRNN(nn.Module):
     [batchSize, timeSteps, inputDims]
     '''
 
-    def __init__(self, cell: RNNCell, batch_first=False):
+    def __init__(self, cell: RNNCell, batch_first=False, cell_reverse: RNNCell=None, bidirectional=False):
         super(BaseRNN, self).__init__()
-        self._RNNCell = cell
+        self.RNNCell = cell 
         self._batch_first = batch_first
+        self._bidirectional = bidirectional
+        if cell_reverse is not None:
+            self.RNNCell_reverse = cell_reverse
+        elif self._bidirectional:
+            self.RNNCell_reverse = cell
```

Comment:
Try to use if cell_reverse is None: first and then else.

```
if cell_reverse is None:
    self.RNNCell_reverse = cell
elif self._bidirectional:
    self.RNNCell_reverse = cell_reverse
```

Reply:
@oindrilasaha please have a look

Reply:
the change suggested is also fine

Reply:
Done, but not sure about logical equivalence. 
@oindrilasaha can you confirm once , if the suggestion is logically equivalent to the original

Reply:
My bad, I read it wrong. i thought it was an else and not life. revert this change

Reply:
Keeping it the original one only finally.